src/models/linePlusBarChart.js

```diff
@@ -600,6 +627,13 @@ nv.models.linePlusBarChart = function() {
         }}
     });
 
+    chart.toggleAxis = function() {
+        switchYAxisOrder = !switchYAxisOrder;
+        var tickFormat = y1Axis.tickFormat();
+        y1Axis.tickFormat(y2Axis.tickFormat());
+        y2Axis.tickFormat(tickFormat);
+    }
+
```

Comment:
why is this here?   The example html page should use the switchYAxisOrder option you added, not require this function on chart that doesn't match how any other options are done within nvd3. 

Other than that it looks good, I'll merge it in if you fix this.